packages/scripts/index.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
+const chalk = require('chalk');
+const fs = require('fs');
+const loadConfigFile = require('rollup/dist/loadConfigFile');
+const ora = require('ora');
```

Comment:
check out `listr2` instead of `ora`  

I will message you with an internal repo that shows it in use

Reply:
I'm going to handle that in a separate PR after playing with `listr2` a bit.